thorlcr/mfilemanager/thmfilemanager.cpp

```diff
@@ -504,39 +502,50 @@ class CFileManager : public CSimpleInterface, implements IThorFileManager
             if (nonLocalIndex)
                 ++total;
             StringBuffer dir;
+            bool dirPerPart = false;
             if (temporary && !job.queryUseCheckpoints()) 
                 dir.append(queryTempDir(false));
             else
             {
+                StringBuffer planeDir;
                 // NB: always >= 1 groupNames
-                StringBuffer curDir;
                 ForEachItemIn(gn, groupNames)
                 {
+                    StringBuffer thisPlaneDir;
+                    bool thisDirPerPart = false;
 #ifdef _CONTAINERIZED
                     if (!globals->getPropBool("@_dafsStorage"))
                     {
                         Owned<IStoragePlane> plane = getDataStoragePlane(groupNames.item(gn), true);
-                        curDir.append(plane->queryPrefix());
+                        thisPlaneDir.append(plane->queryPrefix());
+                        thisDirPerPart = plane->queryDirPerPart();
                     }
 #else
-                    if (!getConfigurationDirectory(globals->queryPropTree("Directories"), "data", "thor", groupNames.item(gn), curDir))
-                        makePhysicalPartName(logicalName, 0, 0, curDir, 0, os); // legacy
+                    if (!getConfigurationDirectory(globals->queryPropTree("Directories"), "data", "thor", groupNames.item(gn), thisPlaneDir))
+                        makePhysicalPartName(logicalName, 0, 0, thisPlaneDir, 0, os); // legacy
```

Comment:
@jakesmith this isn't new, but this code is weird.  It is getting the physical directory - which it then reprocesses later

Reply:
yes I know, and noticed when making these last changes, but tried to ignore.
I don't think it will ever hit this line either, or it would have to be a very strange Directories configuration in the environment xml